torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, List, Optional, Tuple
+
+import torch
+from torch.distributed.fsdp._utils import _apply_to_tensors
+from torch.distributed.fsdp.flat_param import FlatParamHandle
+from torch.distributed.utils import _to_kwargs
+
+
+def _wait_for_computation_stream(
```

Comment:
Rename `_wait_for_previous_optim_step()` to `_wait_for_computation_stream()`. See second bullet of https://github.com/pytorch/pytorch/issues/67288.